actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -2,6 +2,9 @@
 
 require "concurrent/map"
 require "action_view/path_set"
+require "action_view/render_parser"
+require "action_view/ripper_tracker"
+require "action_view/erb_tracker"
```

Comment:
I feel weird requiring this here since it isn't used, but I have to have ERBTracker act like it did before this change. Let me know if that assumption is wrong.